scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Split.scala

```diff
@@ -26,7 +26,7 @@ case class Split(
     modification: Modification,
     cost: Int,
     tag: SplitTag = SplitTag.Active,
-    indents: Vector[Indent[Length]] = Vector.empty[Indent[Length]],
+    indents: Seq[Indent[_]] = Seq.empty,
```

Comment:
Why was `Length` eliminated?

Reply:
Length is the base type for both `Num` and `StateColumn`, so it's not needed.